### PR TITLE
[Fix] Fix the incorrect labels for training vis_head with combined datasets

### PR DESCRIPTION
**Pull request by Ben-Louis**
## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmpose/models/losses/classification_loss.py [hunk 1]
```diff
@@ -14,15 +14,17 @@ class BCELoss(nn.Module):
         use_target_weight (bool): Option to use weighted loss.
             Different joint types may have different target weights.
         loss_weight (float): Weight of the loss. Default: 1.0.
-        with_logits (bool): Whether to use BCEWithLogitsLoss. Default: False.
+        use_sigmoid (bool, optional): Whether the prediction uses sigmoid
+            of softmax. Defaults to False.
```
Comment (Tau-J) [inline mmpose/models/losses/classification_loss.py:18]:
```suggestion
        use_sigmoid (bool, optional): Whether the prediction uses sigmoid
            before output. Defaults to False.
```
Shall we use this kind of description?

**Comment by Tau-J** [comment]
Have you checked the influence on `browse_dataset.py` ?

**Comment by Ben-Louis** [comment]
> Have you checked the influence on `browse_dataset.py` ?

Yes, I tried to browse the combined dataset body7, and the results seem reasonable

**Review by Tau-J — APPROVED** [review]

File: mmpose/models/heads/hybrid_heads/vis_head.py [hunk 2]
```diff
@@ -29,10 +29,10 @@ class VisPredictHead(BaseHead):
 
     def __init__(self,
                  pose_cfg: ConfigType,
+                 override_bbox_score: bool = False,
```
Comment (Tau-J) [inline mmpose/models/heads/hybrid_heads/vis_head.py:32]:
```suggestion
```

**Review by Tau-J — APPROVED** [review]
